### PR TITLE
Add man page for jack_mixer as ReST format source

### LINKED ISSUE #61: Add man page
[Debian has made one years ago](https://salsa.debian.org/multimedia-team/jack-mixer/-/blob/master/debian/jack_mixer.1). It probably needs to be updated before getting it in here (upstream).

### PR DESCRIPTION
**Pull request by SpotlightKid**
Fixes #61.

### CODE REVIEW

File: jack_mixer.1.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+=============
+
+----------------------------------------------
+A graphical multi-channel audio mixer for JACK
+----------------------------------------------
+
+:Author: Nedko Arnaudov <nedko@arnaudov.name> (original author),
+    Frédéric Peters <fpeters@0d.be> (current maintainer).
+:Date: 2020-08-29
+:Copyright: GNU General Pulic License Version 2
```
Comment (trebmuh) [inline jack_mixer.1.rst:12]: typo: Pulic -> Public

File: jack_mixer.1.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+jack_mixer.py [-h] [-c FILE] [-d] [NAME]
+
+
+DESCRIPTION
+===========
+
+This manual page documents briefly the **jack_mixer** command.
+
+**jack_mixer** is multi-channel audio mixer application with a graphical user
```
Comment (trebmuh) [inline jack_mixer.1.rst:29]: forgotten "a" : jack_mixer** is **a** multi-channel...

**Review by trebmuh — COMMENTED** [review]
LGTM

File: jack_mixer.1.rst (hunk 1 above)
Comment (SpotlightKid) [reply jack_mixer.1.rst:12]: Huh, I wonder where I got that from? This is something I would normally copy and paste. But I can't find the same error elsewhere in the repo.

Comment (SpotlightKid) [reply jack_mixer.1.rst]: Fixed

**Comment by SpotlightKid** [comment]
> A mention to NSM in the DESCRIPTION paragraph would be great.

This is not meant to be complete yet. Thatswhy I added the "help wanted" label to issue #61.

I would like the general description to be fleshed out more and at least have a basic explanation of how to use the `jack_mixer` UI. Not in the sense of "click here, then there, etc.", but how to set up a basic mixer configuration.

But I'm not very good at writing documentation, because I get bogged down in detail too much. I'm better at refining existing text. So I would love if somebody wants to write a draft.

**Comment by SpotlightKid** [comment]
I added an initial description of the GUI functions to the man page.

Comments, addition and corrections welcome!

**Comment by SpotlightKid** [comment]
Integrated man page into meson build setup.

`meson dist` will add the pre-generated man page to the source dist, so `rst2man` (from docutils) is not needed to install jack_mixer from the source dist. It is only needed when installing from a Git repo checkout or for creating the source dist.

**Comment by trebmuh** [comment]
nice

**Pull request merged**